Weather_Display_Matrix/openweather_graphics.py

```diff
@@ -76,6 +82,20 @@ def __init__(
             x=0,
             y=0,
         )
+        # # CircuitPython 7+ compatible
+        # icons = displayio.OnDiskBitmap(icon_spritesheet)
+        # self._icon_sprite = displayio.TileGrid(
+        #             icons,
+        #             pixel_shader=icons.pixel_shader,
+        #             width=1,
```

Comment:
`x`, `y`, `width`, `height` & `default_tile` can all be removed as they are the default values.

Reply:
Thank you, removed them with the latest commit.